README.md

```diff
@@ -24,12 +24,11 @@ rbenv install 2.3.0
 rbenv global 2.3.0
 ```
 
-Next, you'll need to make sure that you have PostgreSQL, Redis, Memcached, and Elasticsearch installed. This can be
-done easily on OSX using [Homebrew](http://brew.sh)
+Next, you'll need to make sure that you have PostgreSQL, Redis, Memcached, and Elasticsearch installed. This can be done easily :
+* on OSX using [Homebrew](http://brew.sh) : `brew install postgresql redis memcached elasticsearch`
+* on Linux : `apt-get install postgresql redis-server memcached`. For Elasticsearch, follow the instructions on [their website](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-repositories.html).
```

Comment:
Can we change these lines to look something like this

- For OSX using [Homebrew](http://brew.sh) : `brew install postgresql redis memcached elasticsearch`
- For Linux : `apt-get install postgresql redis-server memcached`. In order to use Elasticsearch follow the instructions on [their website](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-repositories.html).